devops/scripts/benchmarks/benches/compute.py

```diff
@@ -578,14 +589,16 @@ def description(self) -> str:
         dst_type = "device" if self.dstUSM == 1 else "host"
         events = "with" if self.useEvents else "without"
         copy_offload = "with" if self.useCopyOffload else "without"
+        with_barrier = "with" if self.useBarrier else "without"
```

Comment:
nit: The group name would be more concise if you would either add "with barrier" or don't add this string otherwise.

Reply:
this is only for the description (which doesn't have to be very concise) + I wanted to keep it consistent.